### DISCUSSION TITLE
Fine-tunning a molfeat featurizer

### DISCUSSION THREAD

**Discussion by ethancohen123** [datamol-io/molfeat#52] [General]
Hi there, 
I was wondering is it was possible to finetune molfeat featurizer ?
Thanks !

**Comment by cwognum** [comment]
Hi @ethancohen123, 

While this was not the main goal in designing `molfeat`, it is definitively possible! We have actually published a tutorial on how to do this with :hugs: transformers: [Transformer finetuning](https://molfeat-docs.datamol.io/stable/tutorials/transformer_finetuning.html). Let me know if that helps!

**Comment by ethancohen123** [comment]
Awesome thanks !

**Reply by maclandrol** [reply]
@ethancohen123, adding utilities to make it easier to finetune models (inside your favorite pytorch framework) is a short-term goal on our roadmap.
